odin/adapters/adapter.py

```diff
@@ -207,3 +208,18 @@ def wrapper(_self, path, request):
             return func(_self, path, request)
         return wrapper
     return decorator
+
+def wants_metadata(request):
+
+    wants_metadata = False
+
+    if "Accept" in request.headers:
+        accept_elems = request.headers["Accept"].split(';')
+        if len(accept_elems) > 1:
+            for elem in accept_elems[1:]:
+                if '=' in elem:
+                    elem = elem.split('=')
+                    if elem[0].strip() == "metadata":
+                        wants_metadata = str(elem[1]).strip().lower() == 'true'
+
+    return wants_metadata
```

Comment:
needs newline at end of file

Reply:
@MattTaylorDLS Thanks for spotting that. Now fixed